core/src/main/java/io/grpc/internal/CallCredentialsApplyingTransportFactory.java

```diff
@@ -144,7 +145,20 @@ public Attributes getTransportAttrs() {
             }
           };
         try {
-          creds.applyRequestMetadata(requestInfo, appExecutor, applier);
+          // Hack to allow appengine to work when using AppEngineCredentials (b/244209681)
+          // since processing must happen on a specific thread
```

Comment:
Add a period at the end since the comment continues. Or add a newline, or such.

Reply:
Done.